ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryIQv2FunctionalTest.java

```diff
@@ -688,7 +689,7 @@ private static void makeAdminRequest(final String sql) {
       final TestKsqlRestApp target,
       final String sql
   ) {
-    final KsqlRestClient ksqlRestClient = target.buildKsqlClient(validCreds());
+    final KsqlRestClient ksqlRestClient = target.buildKsqlClient(validCreds(), ConsistencyLevel.MONOTONIC_READS);
     final ImmutableMap.Builder<String, Object> builder = new ImmutableMap.Builder<String, Object>()
         .put(KsqlRequestConfig.KSQL_DEBUG_REQUEST, true)
         .put(KSQL_QUERY_PULL_CONSISTENCY_OFFSET_VECTOR_ENABLED, true);
```

Comment:
I think you are seeing an old version of the code. This uses the new consistency level now

Reply:
Yea, I meant whether we need KSQL_QUERY_PULL_CONSISTENCY_OFFSET_VECTOR_ENABLED to be a boolean or if it could be a level. But we don't need to discuss this as part of this PR.

Reply:
Ah gotcha! I guess we can make this change but we can do it when we have more levels than two. It would require also changing the server flag since I am using the same.